framework/schema.py

```diff
@@ -82,6 +92,36 @@ def resolve_lastSeenInLab(self, info):
             return None
 
 
+class UserType(DjangoObjectType):
+    class Meta:
+        model = get_user_model()
+        exclude = ('password',)
+
+
+class CreateUser(graphene.Mutation):
+    user = graphene.Field(UserType)
+
+    class Arguments:
+        username = graphene.String(required=True)
+        password = graphene.String(required=True)
+        email = graphene.String(required=True)
+
+    def mutate(self, info, username, password, email):
+        user = info.context.user
+        if user.is_superuser:
```

Comment:
Allow everyone to create an account and default state should be inactive. Create a new query to get all the inactive users list, only for the superuser. Add a mutation where superusers can activate user accounts.